.github/workflows/linters.yml

```diff
@@ -0,0 +1,27 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Linter
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+     
+      - uses: actions/checkout@v3
+      - name: Super-Linter
+        uses: github/super-linter@v4.10.0
+
```

Comment:
- [ ] You did an amazing job configuring linters :+1: on your repo but as a general requirement for the project you shouldn't custom changes to the linter config and also you shouldn't add linters.yml twice on your repo. If you are having problems with the linters configuration you can always read more [here](https://github.com/microverseinc/linters-config/tree/master/html-css)